crates/cli/src/bin/entryfilelist-test.rs

```diff
@@ -0,0 +1,57 @@
+use std::{error::Error, path::PathBuf};
+
+use clap::Parser;
+use fstools_dvdbnd::{DvdBnd, FileKeyProvider};
+use fstools_formats::entryfilelist::EntryfilelistContainer;
+
+#[derive(Parser, Debug)]
+#[command(version, about, long_about = None)]
```

Comment:
Could you move this to a `describe` command in the new CLI crate?

Reply:
Done per 3c9d315308f0fbc7d836d45e6535543734e5b6cf